.vscode/launch.json

```diff
@@ -7,10 +7,15 @@
     {
       "type": "node",
       "request": "launch",
-      "name": "Launch Insomnia",
+      "name": "Launch Symon",
       "skipFiles": ["<node_internals>/**"],
       "program": "${workspaceFolder}\\bin\\run_dev",
-      "args": ["-I", "./insomnia.json"],
+      "args": [
+        "--symonUrl",
+        "http://localhost:3000/",
+        "--symonKey",
+        "dc5b4caa-46f1-435f-9e23-7141b9b1a85a"
```

Comment:
What will happen if other developer use vscode to run Monika?
Do they need to run symon in their localhost with this `symonKey`?

Reply:
updated @gsukanto